nixos/doc/manual/development/releases.xml

```diff
@@ -203,6 +203,30 @@
   </section>
 </section>
 
+<section xml:id="release-managers">
+  <title>Release Management Team</title>
+  <para>
+    For each release there are two release managers. After each release the
+    release manager having managed two releases steps down and the release
+    management team of the last release appoints a new release manager.
+  </para>
+  <para>
+    This makes sure a release management team always consists of one release
+    manager who already has managed one release and one release manager being
+    introduced to their role, making it easier to pass on knowledge and
+    experience.
+  </para>
+  <para>
+    A release manager's role and responsibilities are:
+  </para>
+  <itemizedlist>
+    <listitem><para>manage the release process</para></listitem>
```

Comment:
I'm not a native speaker, but `a [...] manager [...] manages` is an awkward expression, right?
Instead I'd just suggest to "steal" the bullet points from @fpletz's slides (if he's fine with it :p): https://youtu.be/fdj9tzRaLn4?list=PLgknCdxP89ReQzhfKwMYjLdwWsc7us8ns&t=188 as they were a good summary :-)